stac/Cargo.toml

```diff
@@ -14,6 +14,7 @@ categories = ["science", "data-structures"]
 gdal = ["dep:gdal", "dep:gdal-sys", "dep:log"]
 geo = ["dep:geo"]
 reqwest = ["dep:reqwest"]
+wkb = ["dep:wkb", "geo", "dep:log"]
```

Comment:
FYI the `wkb` crate is AGPL-3 licensed https://docs.rs/wkb/latest/wkb/ and https://github.com/amandasaurus/rust-wkb. Geozero has its own WKB implementation

Reply:
Ah, good to know — I'll bump over to **geozero** as it seems more in-ecosystem. Thanks for the heads up!